src/Shorttag/DownloadUrl.php

```diff
@@ -1,19 +1,19 @@
 <?php
 
-namespace GFExcel\Shorttag;
+namespace GFExcel\Shortcode;
```

Comment:
This is a breaking change, and the path no longer resolves via PSR-4. So we can't do this.

Reply:
Added alias to keep the old path for BC.